### PR TITLE
Add missing US Census GeoID concordances from FIPS

### PR DESCRIPTION
**Pull request by nvkelso**
Related to https://github.com/whosonfirst-data/whosonfirst-data/issues/2135

### CODE REVIEW

File: data/102/080/273/102080273.geojson [hunk 1]
```diff
@@ -93,10 +95,11 @@
     ],
     "wof:breaches":[],
     "wof:concordances":{
-        "hasc:id":"PR.CU"
+        "hasc:id":"PR.CU",
```
Comment (stepps00) [inline data/102/080/273/102080273.geojson:98]: nice!

**Comment by stepps00** [comment]
Same question around geoid value types, but LGTM otherwise.

File: data/102/080/273/102080273.geojson [hunk 2]
```diff
@@ -117,12 +120,12 @@
         "spa",
         "eng"
     ],
-    "wof:lastmodified":1566600191,
+    "wof:lastmodified":1684438911,
     "wof:name":"Culebra",
-    "wof:parent_id":85687331,
+    "wof:parent_id":"85633729",
```
Comment (nvkelso) [inline data/102/080/273/102080273.geojson:125]: Oops, parent_id should be an int!

Comment (nvkelso) [reply data/102/080/273/102080273.geojson]: Fixed by https://github.com/whosonfirst-data/whosonfirst-data-admin-pr/pull/9/commits/a069b8b6c49c7aa471c0c592d4c2ce543bfa1832.

**Pull request merged**